cvmfs/cvmfs.cc

```diff
@@ -2643,7 +2643,10 @@ static int Init(const loader::LoaderExports *loader_exports) {
     download::DownloadManager::kSetProxyBoth);
   g_external_download_ready = true;
   if (use_geo_api) {
-    cvmfs::external_download_manager_->ProbeGeo();
+    std::vector<std::string> host_chain;
```

Comment:
Ah, I think the problem is that if external is the same as the normal urls, there is no need to do the sorting twice.  I.e., if `CVMFS_EXTERNAL_URL`, the external download manager should just apply the already sorted list from the normal one.